### PR TITLE
Add's capability to retrieve members of a group.

### PR DESCRIPTION
**Pull request by denvenafi**
Issue 83

### CODE REVIEW

File: VenafiPS/Public/Get-TppIdentity.ps1 [hunk 1]
```diff
@@ -56,6 +64,10 @@ https://docs.venafi.com/Docs/current/TopNav/Content/SDK/WebSDK/r-SDK-GET-Identit
 
 .LINK
 https://docs.venafi.com/Docs/current/TopNav/Content/SDK/WebSDK/r-SDK-POST-Identity-GetAssociatedEntries.php
+
+.LINK
+https://https://docs.venafi.com/Docs/21.4SDK/TopNav/Content/SDK/WebSDK/r-SDK-POST-Identity-GetMembers.php
```
Comment (gdbarron) [inline VenafiPS/Public/Get-TppIdentity.ps1:69]:
```suggestion
https://docs.venafi.com/Docs/current/TopNav/Content/SDK/WebSDK/r-SDK-POST-Identity-GetMembers.php
```

File: VenafiPS/Public/Get-TppIdentity.ps1 [hunk 2]
```diff
@@ -11,6 +11,9 @@ The individual identity, group identity, or distribution group prefixed universa
 .PARAMETER IncludeAssociated
 Include all associated identity groups and folders
 
+.PARAMETER GetMembers
```
Comment (gdbarron) [inline VenafiPS/Public/Get-TppIdentity.ps1:14]: to keep with existing naming, can you please rename to IncludeMember?

File: VenafiPS/Public/Get-TppIdentity.ps1 [hunk 3]
```diff
@@ -119,6 +134,12 @@ function Get-TppIdentity {
                         $response | Add-Member @{ 'Associated' = $associated.Identities }
                     }
 
+                    if (($response.IsGroup) -and ($GetMembers))  {
+                        $params.UriLeaf = 'Identity/GetMembers'
+                        $params.Body.Add("ResolveNested","1");
```
Comment (gdbarron) [inline VenafiPS/Public/Get-TppIdentity.ps1:139]: do we always want to do this or should it be a param?
also...
```suggestion
                        $params.Body.ResolveNested = '1'
```

File: VenafiPS/Public/Get-TppIdentity.ps1 [hunk 4]
```diff
@@ -119,6 +134,12 @@ function Get-TppIdentity {
...
+                        $members = Invoke-VenafiRestMethod @params
+                        $response | Add-Member @{ 'Members' = $members.Identities}
```
Comment (gdbarron) [inline VenafiPS/Public/Get-TppIdentity.ps1:141]:
```suggestion
                        $response | Add-Member @{ 'Member' = $members.Identities}
```

Comment (gdbarron) [inline VenafiPS/Public/Get-TppIdentity.ps1:141]: Member should have the same properties as the group, ID, IsGroup, Member, Name, Path.

**Review by gdbarron — CHANGES_REQUESTED** [review]

File: VenafiPS/Public/Get-TppIdentity.ps1 (hunk 3 above)
Comment (denvenafi) [reply VenafiPS/Public/Get-TppIdentity.ps1:139]: I'm torn on this one.  I would like to  keep this simple as possible for testing and to implement asap.  We could always add a defaulted parameter later on, I have  yet  to see a  use case to not set ResolveNested .

File: VenafiPS/Public/Get-TppIdentity.ps1 (hunk 4 above)
Comment (denvenafi) [reply VenafiPS/Public/Get-TppIdentity.ps1:141]: It is an array with a number of members, shouldn't it be plural ?

Comment (denvenafi) [reply VenafiPS/Public/Get-TppIdentity.ps1:141]: Pretty sure it does, I  am just returing whatever the GetMembers method returns and it does contain that data.

**Pull request closed without merging**